sdk/python/tests/test_cli_gcp.py

```diff
@@ -53,3 +53,36 @@ def test_basic() -> None:
 
         result = runner.run(["teardown"], cwd=repo_path)
         assert result.returncode == 0
+
+
+@pytest.mark.integration
+def test_fail() -> None:
```

Comment:
Nitpick, but `test_fail` isn't descriptive. If this test fails in CI then we won't have a clue as to what went wrong. Can we use a more precise description so that we know what the scope of this test is?

Reply:
Yes definitely will do.

Reply:
I think the actual function needs to be renamed btw.